airflow/operators/subdag.py

```diff
@@ -115,14 +115,9 @@ def _validate_pool(self, session):
                 pool = session.query(Pool).filter(Pool.slots == 1).filter(Pool.pool == self.pool).first()
                 if pool and any(t.pool == self.pool for t in self.subdag.tasks):
                     raise AirflowException(
-                        'SubDagOperator {sd} and subdag task{plural} {t} both '
-                        'use pool {p}, but the pool only has 1 slot. The '
-                        'subdag tasks will never run.'.format(
-                            sd=self.task_id,
-                            plural=len(conflicts) > 1,
-                            t=', '.join(t.task_id for t in conflicts),
-                            p=self.pool,
-                        )
+                        f"SubDagOperator {self.task_id} and subdag task{len(conflicts) > 1} "
+                        f"{', '.join(t.task_id for t in conflicts)} both use pool {self.pool}, "
+                        f"but the pool only has 1 slot. The subdag tasks will never run."
```

Comment:
Wait this `task{len(conflicts) > 1}` does not look right. It seem it’s wrong since a long time ago but let’s fix it.

Reply:
:) - we can like fix it in separate PR